extension/src/time_weighted_average.rs

```diff
@@ -399,7 +399,7 @@ extension_sql!(
         combinefunc = time_weight_combine,\n\
         serialfunc = time_weight_trans_serialize,\n\
         deserialfunc = time_weight_trans_deserialize,\n\
-        parallel = restricted\n\
+        parallel = unsafe\n\
```

Comment:
I think you should also set `PARALLEL UNSAFE` for all internal functions:
* time_weight_trans
* time_weight_final
* time_weight_combine
* time_weight_trans_serialize
* time_weight_trans_deserialize

Reply:
I think the functions that make up the aggregates should still be parallel safe:

https://github.com/timescale/timescaledb-toolkit/blob/d895f628334769f852979df4a5ca6b2102487934/extension/src/time_weighted_average.rs#L173